src/peerbook/libp2p_peerbook.erl

```diff
@@ -475,10 +482,7 @@ handle_info(notify_timeout, State=#state{}) ->
 handle_info(gossip_peers_timeout, State=#state{peerbook=Handle, tid=TID}) ->
     %% A peer is eligilble for gossiping if it is dialable and has a
     %% minimum number of outbound connections.
-    IsEligibleGossipPeer = fun(Peer) ->
```

Comment:
this is a potentially questionable decision.  If it's not a bad one, mabye we should use `random` here instead of `fold_peers`?

Reply:
Yeah.. by not knowing how connected the peer is you could be picking some really bad counterparts which would not help gossip.